src/playground/sample_data_tooling/sample_data_plugins/ad_plugin_data_config/data_trend_functions.py

```diff
@@ -0,0 +1,166 @@
+from datetime import datetime, timedelta
```

Comment:
Please add unit tests for each py file. I didn't see unit tests for this. Thanks

Reply:
Done, unit tests added to all py files